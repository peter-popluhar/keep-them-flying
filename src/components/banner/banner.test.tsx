import { render, screen } from "@testing-library/react";
import Banner from "./banner";

test("renders Banner", () => {
  const { container } = render(<Banner />);
  const text = screen.getByText(/Keep them flying/i);
  expect(text).toBeInTheDocument();
  expect(container).toMatchSnapshot();
});
