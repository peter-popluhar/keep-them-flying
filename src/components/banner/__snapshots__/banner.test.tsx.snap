// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders Banner 1`] = `
<div>
  <div
    class="banner"
  >
    <h1
      class="heading"
    >
      Keep them flying...
    </h1>
  </div>
</div>
`;
